web/packages/design/src/theme/themes/bblpTheme.ts

```diff
@@ -0,0 +1,224 @@
+      default: '#FFA028',
+      hover: '#FFB04C',
+      active: '#DB8922',
+    },
+
+    secondary: {
+      default: 'rgba(255,255,255,0.07)',
+      hover: 'rgba(255,255,255,0.13)',
+      active: 'rgba(255,255,255,0.18)',
+    },
+
+    border: {
+      default: 'rgba(255,255,255,0)',
+      hover: 'rgba(255, 255, 255, 0.07)',
+      active: 'rgba(255, 255, 255, 0.13)',
+      border: 'rgba(255, 255, 255, 0.36)',
+    },
+
+    warning: {
+      text: '#FFFFFF',
+      default: '#E51E3C',
+      hover: '#FD2D4A',
+      active: '#C31834',
+    },
+
+    trashButton: {
+      default: 'rgba(255, 255, 255, 0.07)',
+      hover: 'rgba(255, 255, 255, 0.13)',
+    },
+
+    link: {
+      default: '#66ABFF',
+      hover: '#99C7FF',
+      active: '#2B8EFF',
+    },
+  },
+
+  tooltip: {
+    background: '#212B2F',
+  },
+
+  progressBarColor: '#00BFA5',
+
+  error: {
+    main: '#E51E3C',
+    hover: '#FD2D4A',
+    active: '#C31834',
+  },
+
+  warning: {
+    main: '#FA5A28',
+    hover: '#FB754C',
+    active: '#D64D22',
+  },
+
+  notice: {
+    background: '#282828', // elevated
+  },
+
+  action: {
+    active: '#FFFFFF',
+    hover: 'rgba(255, 255, 255, 0.1)',
+    hoverOpacity: 0.1,
+    selected: 'rgba(255, 255, 255, 0.2)',
+    disabled: 'rgba(255, 255, 255, 0.3)',
+    disabledBackground: 'rgba(255, 255, 255, 0.12)',
+  },
+
+  terminal: {
+    foreground: '#F1F2F4',
+    background: '#0C143D', // sunken
+    selectionBackground: 'rgba(255, 255, 255, 0.18)',
+    cursor: '#FFF',
+    cursorAccent: '#0C143D',
+    red: dataVisualisationColors.primary.abbey,
+    green: dataVisualisationColors.primary.caribbean,
+    yellow: dataVisualisationColors.primary.sunflower,
+    blue: dataVisualisationColors.primary.picton,
+    magenta: dataVisualisationColors.primary.purple,
+    cyan: dataVisualisationColors.primary.cyan,
+    brightWhite: lighten('#0C143D', 0.89),
+    white: lighten('#0C143D', 0.78),
+    brightBlack: lighten('#0C143D', 0.61),
+    black: '#000',
+    brightRed: dataVisualisationColors.tertiary.abbey,
+    brightGreen: dataVisualisationColors.tertiary.caribbean,
+    brightYellow: dataVisualisationColors.tertiary.sunflower,
+    brightBlue: dataVisualisationColors.tertiary.picton,
+    brightMagenta: dataVisualisationColors.tertiary.purple,
+    brightCyan: dataVisualisationColors.tertiary.cyan,
+  },
+
+  editor: {
+    abbey: dataVisualisationColors.tertiary.abbey,
+    purple: dataVisualisationColors.tertiary.purple,
+    cyan: dataVisualisationColors.tertiary.cyan,
+    picton: dataVisualisationColors.tertiary.picton,
+    sunflower: dataVisualisationColors.tertiary.sunflower,
+    caribbean: dataVisualisationColors.tertiary.caribbean,
+  },
+
+  link: '#66ABFF',
+  success: '#00BFA5',
+
+  dataVisualisation: dataVisualisationColors,
+
+  ...sharedColors,
+};
+
+const theme = {
+  name: 'bblp',
+  // This field should be either `light` or `dark`. This is used to determine things like which version of logos to use
```

Comment:
Would be nice (but not required) if you could enforce this with the type system so that you don't have to put the same comment in multiple places.